src/main/java/pl/betoncraft/betonquest/compatibility/Compatibility.java

```diff
@@ -144,6 +144,10 @@ public Compatibility() {
 			plugin.registerObjectives("npckill", NPCKillObjective.class);
 			plugin.registerObjectives("npcinteract", NPCInteractObjective.class);
 			plugin.registerEvents("movenpc", NPCMoveEvent.class);
+
+			//Register Citizens Events
+			Bukkit.getServer().getPluginManager().registerEvents(new CitizensParticle(), Bukkit.getPluginManager().getPlugin("BetonQuest"));
```

Comment:
Since `CitizensParticle` doesn't have to be a listener (it's a `BukkitRunnable`) this line here isn't needed at all.